package com.thinkinginjava.error_handling_with_exceptions.example.chapter12_6.ex12_6_1;

/**
 * page 257
 *   -- chapter_12 通过异常处理错误
 *
 *   12.6.1 栈轨迹
 */